src/film.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Implementación de clase Film
+ */
+
+const genresEnum = require("../models/genresEnum.js");
+const ValidationError = require("../error/validationError.js");
+
+const minYear = 0;
+const maxYear = 2023;
+
+class Film
+{
+    /**
+     * Metodos
+     * @param {String} title
+     * @param {Number} year
+     * @param {String} description 
+     * @param {Array} genres
+     * @param {Array} actors
+     */
+    constructor(title,year,description,genres,actors)
+    {
+        this.title = title;
+        this.year = year;
+        this.description = description;
```

Comment:
¿Qué HU/issue pide una descripción?
¿Es posible que el año tenga un valor por defecto y por tanto se pueda poner aquí? ¿habéis hecho un issue para esto?

Reply:
Como dijo Groucho Marx, responde primero a la segunda pregunta.
Si hay un issue que dice que, en general, se van a añadir pelis según salgan el año por defecto será el actual que se puede sacar fácilmente de la fecha del sistema. Así puedes crear una peli con un solo parámetro.

Reply:
Ahh entiendo